AWS/legos/aws_get_all_service_names/aws_get_all_service_names.json

```diff
@@ -1,11 +1,11 @@
 {
-  "action_title": "AWS_Get_All_Service_Names_v2.1",
-  "action_description": "This retrieves a list of all AWS services in a region.",
+  "action_title": "AWS Get All Service Names v3",
+  "action_description": "Get a list of all service names in a region",
   "action_type": "LEGO_TYPE_AWS",
-  "action_entry_function": "aws_get_all_service_names2",
+  "action_entry_function": "aws_get_all_service_names3",
```

Comment:
The rules that we check are
Lego Directory name should match the python and JSON file name
Action Entry function should be the same as the Lego directory name
A init.py File should exist for every Lego directory
A README.md should be present for every Lego
(Optional) A Screenshot that is referenced in README.md shows the output of the Action

These are rules defined for the Legos.